stock_orderpoint_mto_as_mts/models/stock_warehouse.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class StockWarehouse(models.Model):
+
+    _inherit = "stock.warehouse"
+
+    archive_orderpoints_mto_removal = fields.Boolean(default=False)
+
+    def _get_locations_for_mto_orderpoints(self):
+        return self.mapped("lot_stock_id")
```

Comment:
Can you depend on https://github.com/OCA/stock-logistics-orderpoint/pull/5/files#diff-34ce65f805726da9a9ebaedb97f0d4188d1335149786f5d37cf07008811ea0ddR11 ?

```suggestion
        self.ensure_one()
        return self.default_orderpoint_location_id or self.lot_stock_id
```